NAMESPACE

```diff
@@ -57,6 +61,24 @@ importFrom(parallel,stopCluster)
 importFrom(readr,read_lines)
 importFrom(readr,write_delim)
 importFrom(servr,make)
+importFrom(sf,"st_crs<-")
```

Comment:
Is this an error that it is importing something called `st_crs<-`? Might be worth double-checking that the @importFrom isn't picking up an extra character.

Reply:
Yes, that is a separate function used in creating the edge overlay for the map functionality.